src/classes/GE_PaymentServices.cls

```diff
@@ -139,8 +148,8 @@ public with sharing class GE_PaymentServices {
     public with sharing class JWTPayload {
         String sfdoId;
 
-        public JWTPayload(GE_PaymentServices serviceInstance) {
-            sfdoId = serviceInstance.config.get(serviceInstance.PRODUCTID);
+        public JWTPayload(String sfdoId) {
```

Comment:
Why is the sfdoId the one parameter we are passing to the class?

Reply:
Right now this is the only config value that this class uses.  If we eventually read product or roles from the passed-in configuration then we can add those params to the constructor.